foundations/javascript_basics/arrays_and_loops.md

```diff
@@ -72,4 +72,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- Some useful array methods - https://www.youtube.com/watch?v=R8rmfD9Y5-c
```

Comment:
Nit: This can be more descriptive and use markdown link syntax:

```suggestion
- Watch [8 Must Know JavaScript Array Methods](https://www.youtube.com/watch?v=R8rmfD9Y5-c)
```